storybook/storybook-docs/src/icon-gallery.stories.mdx

```diff
@@ -0,0 +1,446 @@
+import { Icon } from "@amsterdam/design-system-react";
+import {
```

Comment:
You can do `import * as Icons from '@amsterdam/design-system-react-icons'` to import them all, see [the Icon component](https://github.com/Amsterdam/design-system/blob/develop/storybook/storybook-react/src/Icon/Icon.stories.tsx#L7) for an example

Reply:
Can't you do the javascript stuff in a `tsx` file, and import it in the `mdx`? That would get rid of the issues I think?

Reply:
Smart. I still need to ignore a TypeScript warning as `Icons` is not typed.

Reply:
I think you can set the type like this: `(Object.keys(Icons) as Array<keyof typeof Icons>).map(...)`

Reply:
Ah yes, updated.